src/components/SvgMask.js

```diff
@@ -108,52 +110,36 @@ export default function SvgMask({
             />
           </mask>
           <clipPath id="clip-path">
```

Comment:
you should probs prefix the id (and other ids in here) with the file name, `#clip-path` is generic enough that there's a chance of namespace clashing

Reply:
true, i'll revisit this tho, its not currently working with the name change which was odd.